backend/app/core/config.py

```diff
@@ -0,0 +1,25 @@
+import os
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
+from pydantic import BaseSettings, RedisDsn
+
+yaml_settings = None
+
+# Load the config file present in the root directory (backend)
+here = Path().absolute() / "config.yml"
+with open(str(here)) as f:
+    yaml_settings = yaml.safe_load(f)
```

Comment:
Doing this at the module level means the config file is read when the module is imported. It would be better if it was done in a method on the class.

Reply:
Done.